srcpkgs/rooster/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'rooster'
 pkgname=rooster
-version=2.8.6
+version=2.9.0
 revision=1
 build_style=cargo
 hostmakedepends="pkg-config python3"
-makedepends="libX11-devel libXmu-devel libressl-devel libsodium-devel"
+makedepends="libX11-devel libXmu-devel libressl-devel libsodium-devel libssl48"
```

Comment:
Why did you add this?? The correct `libssl` will be pulled in by `libressl-devel`.